tests/newsletters/test_newsletter_views.py

```diff
@@ -243,23 +288,6 @@ def test_skip_inactive(admin, client, project, user_factory, follow_factory):
     assert len(mail.outbox) == 0
 
 
-@pytest.mark.django_db
-def test_access_admin_newsletter(client, project, admin):
-    organisation = project.organisation
-
-    assert organisation.initiators.count() == 1
-    initiator = organisation.initiators.first()
-
-    url = reverse('meinberlin_newsletters:newsletter-create')
-    client.login(username=admin.email, password='password')
-    response = client.get(url)
-    assert response.status_code == 200
-
-    client.login(username=initiator.email, password='password')
-    response = client.get(url)
-    assert response.status_code == 403
-
```

Comment:
I meant this test. But that was only for the weird view from the dropdown...

Reply:
Ah, I moved this to the platform email tests. Coming soon :)